contracts/core/chainlink-oracle/v1/ChainlinkOracle.sol

```diff
@@ -7,39 +7,33 @@ import "external-interfaces/IChainlinkAggregatorV3.sol";
 
 import "core/globals/v1/interfaces/IMapleGlobals.sol";
 
-/// @title ChainlinkOracle is a wrapper contract for Chainlink oracle price feeds that allows for manual price feed overrides.
-contract ChainlinkOracle is Ownable {
+import "./interfaces/IChainlinkOracle.sol";
 
-    IChainlinkAggregatorV3 public priceFeed;
-    IMapleGlobals public globals;
+/// @title ChainlinkOracle is a wrapper contract for Chainlink oracle price feeds that allows for manual price feed overrides.
+contract ChainlinkOracle is IChainlinkOracle, Ownable {
 
-    address public immutable assetAddress;
+    IChainlinkAggregatorV3 public override priceFeed;
+    IMapleGlobals public override globals;
 
-    bool   public manualOverride;
-    int256 public manualPrice;
+    address public override immutable assetAddress;
```

Comment:
Why do we need `override` here? My assumption here is that it is needed to avoid shadow declaration of the same variable name.

Reply:
If we remove `override` we get this type of error:
```
contracts/core/loan/v1/Loan.sol:35:5: TypeError: Overriding public state variable is missing "override" specifier.
    State public loanState;
    ^--------------------^
contracts/core/loan/v1/interfaces/ILoan.sol:90:5: Overridden public state variable is here:
    function loanState() external view returns (State);
    ```

Reply:
Needs to override the interface (function) defined in `IChainlinkOracle.sol`.